models/public/anti-spoof-mn3/model.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  "anti-spoofing-mn3" is an anti-spoofing binary classificator based on the MobileNetv3
+  <https://arxiv.org/abs/1905.02244>, trained on the Celeba_Spoof dataset <https://arxiv.org/abs/2007.12342>.
+  It's a small, light model, trained to predict whether or not a spoof RGB image given
+  to the input. A lot of advanced techniques have been tried and selected the best
+  suit options for the task. For details see original repository <https://github.com/kirillProkofiev/light-weight-face-anti-spoofing>.
+task_type: classification
+files:
+  - name: anti-spoof-mn3.onnx
+    size: 12270179
+    sha256: c4c99af04603b62d7e44f6f4daeb33e0daeccc696008c0b1d62f6f5cebbb3262
+    source:
+      $type: google_drive
+      id: 1KmM2vwGBuKxy7RE9hz1m5D0yyh2N9Amo
+model_optimizer_args:
+  - --input_shape=[1,3,128,128]
+  - --input=actual_input_1
+  - --mean_values=actual_input_1[151.2405,119.5950,107.8395]
+  - --scale_values=actual_input_1[63.0105,56.4570,55.0035]
+  - --reverse_input_channels
+  - --input_model=$dl_dir/anti-spoof-mn3.onnx
+framework: onnx
+quantizable: yes
```

Comment:
@kirillProkofiev please remove quantization status, it will be added after analysis from POT side for selection the best suitable quantization algo